src/Symfony/Component/Process/Process.php

```diff
@@ -437,11 +437,13 @@ public function wait(callable $callback = null)
      * It allows to have feedback from the independent process during execution.
      *
      * @param callable $callback
+     * 
+     * @return bool
```

Comment:
should be removed (same for callback btw) as we remove docblocks that don't add anything on top of the PHP signature.